sites/all/themes/wille/templates/ting/ting-object--full.tpl.php

```diff
@@ -10,41 +10,44 @@
  */
 ?>
 <div class="<?php print $classes; ?> clearfix"<?php print $attributes; ?>>
-  <?php hide($content['group_ting_object_right_column']['group_material_details']); ?>
-  <?php hide($content['group_ting_object_right_column']['ting_relations']); ?>
+  <?php if ('ting_collection' === $bundle): ?>
+    <?php echo render($content); ?>
+  <?php else : ?>
 
-  <?php /*echo render($content);*/ ?>
```

Comment:
Comment out php code?

Reply:
I've removed this code as part of the pull request.